### ISSUE TITLE
Connection timeout while trying to update addon

### ISSUE THREAD

**Issue by crasher02** [WowUp/WowUp#731]
**Describe the bug**

Whenever I try to update or install any addon I get an "error" label on a gray background ( where the update button usually is)
I have tried the latest stable version 2.1.2 and 2.2 as well and the problem still persists.

I have also checked the logs and there seems to be a connection timeout. I don't know if you can harvest
 the logs on your own ( i just uploaded the log files to be sure )
[main.log](https://github.com/WowUp/WowUp/files/5975355/main.log)

**To Reproduce**
1. Go to 'my addons view'
2. Click on 'update / update all'
3. wait a few seconds
4. get Error labeled button ( as you can see in the screenshot below )

**Expected behavior**

**Screenshots**
![image](https://user-images.githubusercontent.com/79001406/107846061-65f08080-6de9-11eb-81c4-4321bb10c246.png)

**Desktop (please complete the following information):**
Edition	Windows 10 Pro
Version	20H2
Installed on	‎12/‎19/‎2020
OS build	19042.804
Experience	Windows Feature Experience Pack 120.2212.551.0

**Additional context**

**Comment by jliddev** [comment]
If you go to Options > Debug > Show Log Files, and send those files we can look into it.

**Comment by crasher02** [comment]
[main.log](https://github.com/WowUp/WowUp/files/5977305/main.log)
[renderer.log](https://github.com/WowUp/WowUp/files/5977306/renderer.log)

Both file attached. I have updated to release 2.2.0 beta.2. Nothing changed unfortunately.

BR,

**Comment by jliddev** [comment]
Not sure what the network issue here is, its timing out connecting to some AWS ip address.

**Comment by crasher02** [comment]
Hello,

Yes I saw that as well. I was only curious to see if you had this problem before or it was reported by anyone else and ofc, what was the workaround.

I am not using any proxy or vpn on the machine. Reinstall didn't work. I've also enabled the FW rules for that speciffic ip and port and I still have the same error.

Actually, I think I will try with a VPN and see whether connection issues still persist. I will check the logs afterwards and come back with a feedback. I have some support experience from early working days and here they come quite handy.
